addons/udes_stock/views/stock_picking.xml

```diff
@@ -101,6 +101,15 @@
                 <attribute name="invisible">1</attribute>
             </xpath>
 
+            <!-- Remove Return button for all -->
+            <xpath expr="//button[@name='%(stock.act_stock_return_picking)d']" position="replace"/>
```

Comment:
I would suggest to make it invisible without condition than to replace.
```suggestion
            <xpath expr="//button[@name='%(stock.act_stock_return_picking)d']" position="attributes">
                <attribute name="invisbile">1</attribute>
            </xpath>
```

Reply:
Armand beat me to it. :)

Reply:
Done